ACPs/204-precompile-secp256r1/README.md

```diff
@@ -27,20 +27,20 @@ The 100x gas cost reduction makes these use cases economically viable while main
 
 ## Specification
 
-This ACP implements [RIP-7212](https://github.com/ethereum/RIPs/blob/master/RIPS/rip-7212.md) for secp256r1 signature verification on Avalanche. The specification follows RIP-7212 exactly, with the precompiled contract deployed at address `0x0200000000000000000000000000000000000006`.
+This ACP implements [EIP-7951](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-7951.md) for secp256r1 signature verification on Avalanche. The specification follows EIP-7951 exactly, with the precompiled contract deployed at address `0x0200000000000000000000000000000000000006`.
 
 ### Core Functionality
 
 - Input: 160 bytes (message hash + signature components r,s + public key coordinates x,y)
 - Output: success: 32 bytes `0x...01`; failure: no data returned
-- Gas Cost: 3,450 gas (based on EIP-7212 benchmarking)
+- Gas Cost: 3,450 gas (based on EIP-7951 benchmarking)
```

Comment:
This is the incorrect gas cost if we're following 7951.

Reply:
updated